lib/customer.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'restaurant'
+
+class Customer
+
+  attr_accessor :order, :total_cost
+
+  def initialize
+    @order = []
+    @total_cost = 0
+  end
+
+  def see_all_dishes(restaurant)
+    restaurant.dishes.each do |item, price|
+      # price = sprintf('%.2f', price)
+      p "#{item} - £#{sprintf('%.2f',price)}"
+    end
+  end
+
+  def select(item, quantity, restaurant)
+    item = item.to_sym
+    if restaurant.dishes.include?(item)
+      price = restaurant.dishes.fetch(item)
+      quantity.times { order << [item.to_s, price] }
+    else
+      fail 'Dish not available'
+    end
+  end
+
+  def current_order
+    if order.empty?
+      "Nothing ordered yet"
+    else
+      order.each { |selection| puts "#{selection[0]}, £#{sprintf('%.2f',selection[1])}" }
```

Comment:
Line is too long. [89/80]<br>Space missing after comma.<br>Do not write to stdout. Use Rails' logger if you want to log.